### ISSUE TITLE
Add ability to provide source object ID in search endpoints

### ISSUE THREAD

**Issue by korikuzma** [cancervariants/metakb#269]
Currently, we run query parameters (variation, disease, therapy, gene) through our normalizers. I think we should allow for source identifiers (such as civic.vid:12) to be searched. I'm not sure if we would want to create a separate endpoint for this or add to existing enpoints

**Comment by jsstevenson** [comment]
Yeah, this is interesting. Since we know the source ID patterns in advance, maybe we could do a quick regex on each term as it comes in to determine whether it gets sent to a normalizer or incorporated directly into a query (optionally disable-able?).

**Comment by korikuzma** [comment]
We'll probably need something like this at some point, but for now I'm going to close. If it is needed, we can re-open or create a new issue that uses the issue templates.
